### PR TITLE
Patch 2019-09-16: New characters, weapons and weapon uncap

### PR DESCRIPTION
**Pull request by Groovman**
### Characters
* [x]  [Colossus](https://gbf.wiki/Colossus)
* [x]  [Sara (Light)](https://gbf.wiki/Sara_(Light))
 
### Weapons
* [x]  [Metal Destroyer](https://gbf.wiki/Metal_Destroyer)
* [x]  [Eon](https://gbf.wiki/Eon)

### Uncaps
* [x]  [Purifying Thunderbolt](https://gbf.wiki/Purifying_Thunderbolt)

## Not implemented
- Colossus
  - [x] Sweltering Armor: 200% charge bar maximum limit.
    (Can only use charge attack when charge bar is at 200%.)

* Please merge in order of date.

### CODE REVIEW

**Comment by kei-gbf** [comment]
- [x] check no more "不詳"
  - Hits 1 row: モルフェとヴェトル (SR)]]|&color(#3366ff){水};|特殊|不明|不詳|
    - [x] Ok  expect PR #355 will update this to male/female

File: armData.json [hunk 1]
```diff
@@ -6720,6 +6720,27 @@
         "imageURL": "./imgs/1040709000.png",
         "en": "Purifying Thunderbolt"
     },
+    "[4凸]悪滅の雷": {
+        "ja": "[4凸]悪滅の雷",
+        "element": "fire",
+        "type": "bow",
+        "skill1": "normalHissatsuL",
+        "skill2": "normalHPL",
```
Comment (kei-gbf) [inline armData.json]: HP?

**Comment by kei-gbf** [comment]
```md
## Not implemented
- Colossus
  - [ ] Sweltering Armor: 200% charge bar maximum limit.
    (Can only use charge attack when charge bar is at 200%.)
```

File: txt_source/armData-ssr.txt [hunk 2]
```diff
@@ -329,6 +329,7 @@
 |&attachref(img/1040708700.png,15%);|太歳精弓|&color(#3366ff){水};|弓|40|[[冥助の破弓>武器奥義#myoujo]]|[[霧氷の攻刃II>スキル一覧#muhyounokoujin2]]||50|420|300|2500|[[フォリア(リミテッドver) (SSR)]]加入|レジェンドガチャ||
 |&attachref(img/1040708700.png,15%);|[4凸]太歳精弓|&color(#3366ff){水};|弓|40|[[冥助の破弓>武器奥義#myoujo]]|[[霧氷の攻刃III>スキル一覧#muhyounokoujin3]]|[[霧氷の技巧>スキル一覧#muhyounogikou]]|50|420|363|3020|[[フォリア(リミテッドver) (SSR)]]加入|レジェンドガチャ|○|300|2500|
 |&attachref(img/1040709000.png,15%);|悪滅の雷|&color(#ff0000){火};|弓|40|[[ダガラハット>武器奥義#dagara]]|[[紅蓮の必殺>スキル一覧#gurennohissatu]]||32|508|242|2800|[[シヴァ (SSR)]]加入|レジェンドガチャ||
+|&attachref(img/1040709000.png,15%);|[4凸]悪滅の雷|&color(#ff0000){火};|弓|40|[[ダガラハット>武器奥義#dagara]]|[[紅蓮の必殺>スキル一覧#gurennohissatu]]|[[紅蓮の守護>スキル一覧#gurennosyugo]]|32|508|295|3373|[[シヴァ (SSR)]]加入|レジェンドガチャ|○|242|2800|
```
Comment (kei-gbf) [inline txt_source/armData-ssr.txt]: 紅蓮の守護 -> 紅蓮の技巧

Comment (Groovman) [reply txt_source/armData-ssr.txt]: Copied the wrong string.

**Pull request merged**